activitysim/core/util.py

```diff
@@ -704,9 +704,9 @@ def drop_unused_columns(
         custom_chooser_lines = inspect.getsource(custom_chooser)
         unique_variables_in_spec.update(re.findall(pattern, custom_chooser_lines))
 
-    logger.info("Dropping unused variables in chooser table")
+    logger.debug("Dropping unused variables in chooser table")
```

Comment:
This is clearly unrelated to this PR.

I do think we should address our logging, by default we are logging far too much